modules/react/collection/lib/ListBox.tsx

```diff
@@ -122,6 +122,7 @@ export const ListBox = createContainer('ul')({
         {...handleCsProp(
           {
             style: {
+              height,
```

Comment:
was this for virtualization ?

Reply:
No. If you set `height`, it doesn't work since it is removed from the prop list, but not added anywhere. It was a bug I noticed